packages/childchain/README.md

```diff
@@ -1,14 +1,39 @@
 # Running the node
 
-To start the node make sure to have ethermint (v0.1.0 or higher) installed and have your GO Path exposed: 
+To start the node make sure to have ethermint (v0.1.0 or higher) installed and have your GO Path exposed:
 
 ```
 git clone https://github.com/ChainSafe/ethermint
 cd ethermint
 make install
 ```
-To run the chain run `./childchain.sh` and in a second terminal/tab `ethermintcli rest-server --laddr "tcp://localhost:8545" --unlock-key mykey --chain-id 8 --trace`
 
 # Run the tests
 
 To compile the contracts run `truffle build`, to run the tests run `truffle test`.
+
+# Start-up script
+
+The underlying bash-Script `childchain.sh` starts the Ethermint chain, initializes a genesis account as well as starting a RPC server. To stop the chain the command `pkill ethermint` should be used or analogous command.
```

Comment:
Some more details on genesis generation would be nice. Also can pregenerated genesis be used? Also see question by @johannbarbie:
> how is the genesis file generated, without exposing the validator private keys to single machine?

Reply:
Template genesis files are created by the deployment server as written below, genesis txs have to be signed by validators.

Reply:
Understood. but it's still not clear for me from this guide if new genesis must be generated on every script run or I can use the one that was generated before.